packages/flame_3d/lib/src/resources/light/lighting_info.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flame_3d/resources.dart';
+
+class LightingInfo {
+  AmbientLight ambient = AmbientLight();
+
+  Iterable<Light> lights = [];
```

Comment:
```suggestion
  final lights = <Light>[];
```

Reply:
also re-assigned

Reply:
Maybe better to clear it and add the new lights instead?

Reply:
I chose to re-assign to avoid iterating over the list every tick.
we also re-assign the other field so it feels consistent

Reply:
removed world config